setup.py

```diff
@@ -13,7 +13,7 @@ def long_description():
 setup(
     name="django-allauth-2fa",
     version="0.7.dev",
-    packages=find_packages(),
+    packages=find_packages('.', include=('allauth_2fa', 'allauth_2fa.*')),
```

Comment:
I suspect this would be clearer as excluding the `tests` package?

Reply:
Technically yes, but I wanted to be explicit about what _is_ included, in case other packages get added to the source root.